src/views/Restoration/RestorationBusinessName.vue

```diff
@@ -43,12 +43,11 @@
       >
         <RestorationType
           id="restoration-type"
-          class="mb-n5"
           :class="{ 'approval-restoration-invalid-section': invalidSectionRestoration }"
         />
 
         <!-- Divider b/w Restoration and Approval type -->
-        <v-divider class="mb-11 mr-5 ml-5"></v-divider>
+        <v-divider class="mb-10 mr-5 ml-5"></v-divider>
```

Comment:
I didn't notice before that this is how it was done, but it's not the right way to do this -- the v-divider isn't the problem, so changing it is not the solution. (This is forming a biding/relationship between the block above the divider and the divider that shouldn't exist.)

The proper solution should have the block above the divider be the correct height.

Reply:
Fixed.